ptxcompiler/patch.py

```diff
@@ -107,6 +107,37 @@ def get_cubin(self, cc=None):
 def patch_needed():
     logger = get_logger()
 
+    # The patch is needed if the user explicitly forced it with an environment
+    # variable.
+    apply = os.getenv("PTXCOMPILER_APPLY_NUMBA_CODEGEN_PATCH")
+    if apply is not None:
+        logger.debug(f"PTXCOMPILER_APPLY_NUMBA_CODEGEN_PATCH={apply}")
+        try:
+            apply = int(apply)
+        except ValueError:
+            apply = False
+
+    if apply:
+        return True
```

Comment:
Maybe:
```suggestion
    if apply is not None:
        logger.debug(f"PTXCOMPILER_APPLY_NUMBA_CODEGEN_PATCH={apply}")
        try:
             return int(apply)
        except ValueError:
             return False
```

Reply:
Ah yes, I see I changed the logic here too. I think we should leave as is. So `APPLY=1` is a hard "definitely apply" and `APPLY=0` is a soft "don't apply": first check if `CHECK=1`